<launch>
    <node pkg="spirecv_ros" type="autofocus_vehicle_detection" name="autofocus_vehicle_detection" output="screen">
        <param name="uav_id" type="int" value="1" />
        <param name="local_saving_path" type="string" value="$(find spirecv_ros)/video" />
    </node>
</launch>
